safe_transaction_service/history/models.py

```diff
@@ -151,6 +165,49 @@ def bulk_create_from_generator(
                 return total
 
 
+class IndexingStatusManager(models.Manager):
+    def get_erc20_721_indexing_status(self) -> "IndexingStatus":
+        try:
+            return self.get(indexing_type=IndexingStatusType.ERC20_721_EVENTS.value)
+        except self.model.DoesNotExist:
+            # Start indexing on master copies minimum block number
+            min_master_copies_block_number = (
```

Comment:
I only want be sure that I'm understanding this correctly. This should be an initial state where still not ERC20/ERC721 transactions, that means we should start looking for them from the minimal `masterCopy` block indexed?

Reply:
That's right, for people setting up the service from scratch. I realise I have to do it on the migration too to follow the same logic, or maybe I can just remove it from here as the migration will guarantee field always exists 👍🏻

Reply:
Done